pcdsdevices/ipm.py

```diff
@@ -25,14 +29,57 @@ class IPMTarget(InOutRecordPositioner):
     _transmission = {st: 0.8 for st in in_states}
 
 
-class IPM(Device):
+class IPMDiode(Device):
+    """
+    Diode of a standard intensity position monitor.
+
+    This device has members for x and y motors, where x-motion is set up as an
+    IMS motor and y-motion is an InOutRecordPositioner that moves the diode
+    position to any of the four set positions, or out. There is also a member,
+    y, which points to the motor of the y-motion.
+    """
+
+    x = FCpt(IMS, '{self.x_prefix}', kind='normal')
```

Comment:
`self.x_prefix` is hard-coded in the init, so there is no need to use `FCpt`
```suggestion
    x = Cpt(IMS, ':XMOTOR', kind='normal')
```

Reply:
Yeah, this was a placeholder while I was deciding whether it should be specified at instantiation or add the alias to the IOC. I'll probably go the IOC route and use this the way you suggested.